Tests/Unit/HttpTestServiceTest.php

```diff
@@ -234,4 +233,31 @@ public function testHttpHeaderComparison()
         $this->assertFalse($stub->checkSingleHeader('http://foo.bar.de/blub/blah', $compare));
         $this->assertFalse($stub->checkSingleHeader('http://foo.bar.de/blubber/bl�h', $compare));
     }
+
+    /**
+     * @dataProvider parseHeaderDateDataProvider
+     * @param mixed $dateString
+     * @param mixed $expectedDate
+     */
+    public function testParseHeaderDate($dateString, $expectedDate)
+    {
+        $subject = new \Caretaker\Caretaker\Tests\Unit\Stubs\HttpTestServiceStub();
+        $this->assertEquals($expectedDate, $subject->parseHeaderDate($dateString));
+    }
+
+    /**
+     * @return array
+     */
+    public function parseHeaderDateDataProvider()
+    {
+        $now = time();
+        return array(
+                array('Tue, 15 Nov 1994 08:12:31 GMT', '784887151'),
```

Comment:
What about strings with different localization? The code before the change was explicitly to convert German strings into compatible ones.

Reply:
> explicitly to convert German strings into compatible ones
I think the previous code dose ONLY work for english date strings, not for German ones.

However RFC1123 does only allow english strings for Weekday and Month. 

As this is about parsing a date send in a HTTP-Response, this should be fine.